lib/stale.js

```diff
@@ -10,28 +10,54 @@ module.exports = class Stale {
 
     await this.ensureStaleLabelExists();
 
-    this.github.paginate(this.getStaleIssues(), res => {
-      res.data.items.filter(issue => !issue.locked)
-                .forEach(issue => this.mark(issue));
-    });
+    this.mark();
+    this.sweep();
+  }
+
+  mark() {
```

Comment:
Ah, so method parameters do not contribute to method signature when it comes to collisions? Interesting!